lib/model/query/auth.js

```diff
@@ -47,6 +51,10 @@ limit 1`)
 const canAssignRole = (actor, role, actee) => ({ Auth }) =>
   Auth.verbsOn(actor.id, actee).then((hasArray) => {
     const has = new Set(hasArray);
+    // `open_form` is subset of `form` so if someone has grant access on `form`
+    // they should be able do it on `open_form` as well
+    if (has.has('form.list')) has.add('open_form.list');
+    if (has.has('form.read')) has.add('open_form.read');
```

Comment:
Rather than inferring `open_form.list` and `open_form.read` from `form.list` and `form.read`, could we add `open_form.list` and `open_form.read` to the list of verbs for admins and project managers? I think that would simplify some of the SQL below as well.